apps/comps/config/private.ts

```diff
@@ -0,0 +1,105 @@
+import z from "zod/v4";
+
+import { SpecificChainSchema } from "@recallnet/services/types";
+
+import {
+  config as publicConfig,
+  configSchema as publicConfigSchema,
+} from "./public";
+
+const configSchema = publicConfigSchema
+  .extend({
+    evmChains: z
+      .array(SpecificChainSchema)
+      .default([
+        "eth",
+        "polygon",
+        "bsc",
+        "arbitrum",
+        "base",
+        "optimism",
+        "avalanche",
+        "linea",
+      ]),
+    specificChainTokens: z
+      .partialRecord(
+        SpecificChainSchema,
+        z.record(z.string().min(1), z.string().min(1)),
+      )
+      .default({}),
+    specificChainBalances: z
+      .partialRecord(
+        SpecificChainSchema,
+        z.record(z.string().min(1), z.number()),
+      )
+      .default({}),
+    watchlist: z.object({
+      chainalysisApiKey: z.string().default(""),
+    }),
+    priceTracker: z.object({
+      maxCacheSize: z.coerce.number().default(10000),
+      priceTTLMs: z.coerce.number().default(60000),
+    }),
+    email: z.object({
+      apiKey: z.string().default(""),
+      mailingListId: z.string().default(""),
+      baseUrl: z.url().default("https://app.loops.so/api/v1"),
+    }),
+    api: z.object({
+      domain: z.url().default("https://api.competitions.recall.network/"),
+    }),
+    security: z.object({
+      rootEncryptionKey: z.string().min(1),
+    }),
+    tradingConstraints: z.object({
+      defaultMinimum24hVolumeUsd: z.coerce.number().default(100000),
+      defaultMinimumFdvUsd: z.coerce.number().default(1000000),
+      defaultMinimumLiquidityUsd: z.coerce.number().default(100000),
+      defaultMinimumPairAgeHours: z.coerce.number().default(168),
+    }),
+    maxTradePercentage: z.coerce.number().min(1).max(100).default(25),
+    rateLimiting: z.object({
+      maxRequests: z.coerce.number().default(100),
+      windowMs: z.coerce.number().default(60000),
+    }),
+    tradingApi: z.object({
+      baseUrl: z.url(),
+      sandboxApiUrl: z.url().optional(),
+      sandboxAdminApiKey: z.string().optional(),
+    }),
+  })
+  .strict();
+
+export const config = {
```

Comment:
I think this is missing actually setting up `specificChainTokens` and `specificChainBalances`

Reply:
Yea I left them default (empty) for now since we're not doing trading apis yet. We can set them for sure.